rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/30_synonyms_delete.yml

```diff
@@ -62,3 +62,49 @@ setup:
           id: "test-other-1"
         - synonyms: "test => check"
           id: "test-other-2"
+
+---
```

Comment:
Should we move this test to modules: analysis-common where we put other similar tests for synonyms?  If we do this, we don't need to include `analysis-common` module.